src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2593,6 +2599,13 @@ static bool deleteContextAttributeItem
       *currentLocAttrName = "";
     }
 
+    /* Check aspects related to date expiration.
+     * If the target attr is date expiration, delete the relative pointer
+     */
+    if (targetAttr->name == ENT_EXPIRATION){
+      *dateExpiration = 0;
```

Comment:
Not sure of the purpose of this... maybe is a way to signal the caller (processContextAttributeVector() function) that the dateExpiration field has to be removed from database?

Reply:
yes, in any case, I'm reviewing all the management of pointers and values, in cases of creation and cancellation, so as to better signal to the caller when a date is not present in a creation request or has been deleted as like in this case.

Reply:
Fixed in 5a4703d